src/locales/en/common.json

```diff
@@ -286,6 +288,10 @@
       "title": "Opps, your device should not be in Bootloader mode",
       "description": "Please restart your device or contact us"
     },
+    "UnavailableTezosOriginatedAccountReceive": {
+      "title": "Cannot receive funds on originated account",
+      "description": "If you want to receive funds, please use the parent account"
+    },
```

Comment:
we'll need to sync these when we have wording

Reply:
yes